src/plugins/legacy/itkDataImage/CMakeLists.txt

```diff
@@ -30,9 +30,33 @@ add_definitions(-D${TARGET_NAME_UP}_VERSION="${${TARGET_NAME}_VERSION}")
 find_package(dtk REQUIRED)
 include_directories(${dtk_INCLUDE_DIRS})
 
-find_package(ITK REQUIRED)
+set(ITK_IMAGES_DEPENDENCIES
+	ITKIOMeta
```

Comment:
line 34, 43 and 46 have indentation problems

Reply:
Ok